### ISSUE TITLE
Validity / Visibility should have an `inverse` attribute

### ISSUE THREAD

**Issue by end2endzone** [end2endzone/ShellAnything#49]
The `Validity / Visibility` xml element in Configuration Files should have an attribute to inverse the logic of the validity/visibility. For instance, inverse the `fileextensions` attribute.

A typical use case is the following:
* A `split file` menu accept only a single file. When executed, it generates multiple files of which the first "part" has the file extension `001`.
* A `join files` menu accept all files that matches `fileextensions="001"`

**Edit:**
To prevent the `split file` menu to be displayed when user selects a *.001 file (we don't need to split a file twice), the menu should also ***not*** accept files that matches `fileextensions="001"`.

In the end, the `split file` menu validity should be:
1. accept only a **single file**.
2. accept files which file extension **is not** `001`.

The validity element should look like this:
```xml
<validity maxfiles="1" maxfolders="0" fileextensions="001" inverse="fileextensions" />
```

**Comment by end2endzone** [comment]
More examples:
* `<validity fileextensions="001" inverse="fileextensions" />`, valid when the file _**is not**_ a *.001 file.
* `<validity maxfiles="1" maxfolders="0" inverse="maxfiles" />`, valid when _**at least**_ one file is selected.
* `<validity properties="process.started" inverse="properties" />`, valid when the property **_is not_** yet specified.
* `<validity exists="${env.USERPROFILE}\config.ini" inverse="exists"  />`, valid only when the user's configuration file **_is not_** yet created.

The `inverse` attribute should be a list (separated with `;` character) to allow insersing multiple attributes.

The attribute should allow the `all` value to inverse all attributes.

**Comment by end2endzone** [comment]
Feature implemented in master branch in commit 455192c1d316d649af698a8c17f84eddaa848b4e.
